frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -140,6 +141,15 @@ def typeof(self, value: Union[ValueDescription, Node]) -> ConversionType:
 
         return result if value.is_scalar else self.tensor(result, value.shape)
 
+    def fork_type(self, type_, bit_width):
+        return self.typeof(
+            ValueDescription(
+                dtype=Integer(is_signed=type_.is_signed, bit_width=bit_width),
+                shape=type_.shape,
+                is_encrypted=type_.is_encrypted,
+            )
+        )
```

Comment:
This should support other properties, not jut bit width. And bit width should be optional to override.

Reply:
Like configuration.fork(...).

Reply:
Agree. My idea was to implement what I need and let the next one in need adding the is_signed param and make them optional. So since there is only bit_width, I didn't want it to be optional.